wazimap_ng/profile/serializers/metrics_serializer.py

```diff
@@ -10,16 +10,22 @@ def get_indicator_data(profile_key_metric, geographies):
     indicator_data = IndicatorData.objects.filter(indicator__profilekeymetrics=profile_key_metric, geography__in=geographies)
     return indicator_data
 
+def indicator_exists(profile_key_metric, geographies):
```

Comment:
that's not a great name. :) 
maybe something like: `get_data_for_key_metric_and_geographies` or so